yetanotherco/aligned_layer
# feat(audit): Use block timestamp instead of block number.

closes: #1069

Switches the userData to check and return in events the block timestamp instead of block number.

## To Test:
- [x] `anvil_deploy_aligned_contracts` and `anvil_start_with_block_time`
- [x] Run local testnet and ensure nothing errors.

# To Test
- Change modify the [block time](https://github.com/yetanotherco/aligned_layer/pull/1085/files#diff-20c3c151edaf33f5a8a50a8dc5945aeda8c89733dc7fc1ca53f63a8b05609585L23) constant. You should observe the unlock time decrease in the local testnet.
